tests/test_reporter.py

```diff
@@ -123,6 +123,7 @@ def setUp(self):
             'krelease': '3.10.0',
             'baserepo': 'git://git.example.com/kernel.git',
             'basehead': '1234abcdef',
+            'kernel_arch': 'x86_64',
```

Comment:
Our current statefiles always have the kernel_arch specified if the build started. However, the tests didn't include that since they haven't been updated in a while.

Reply:
Ok, thanks for the explanation :)